tasks/src/translation/about.md

```diff
@@ -0,0 +1,36 @@
+## Use Case: Information Extraction from Invoices
+Translation models can be used to build conversational agents across different languages. This can be done in two ways.
+One option is to translate the training data of intent classification algorithm and the responses defined for each intent from source language of existing data to target language, and train a new intent classification and dialogue management models. You can proofread the responses in the target language to provide better control over your chatbot’s outputs.
+Another way is to put one translation model from the target language to the language the chatbot is trained on, this will translate the user inputs. The output of this will be input to intent classification algorithm in the source language. After predicting the user intent, we take the classified intent and the response to that intent in the source language. We take the output and translate it to user’s language. This approach might be less reliable since chatbot will output responses that are not defined before.
+
+## Inference
+Translation models are loaded with “translation_xx_to_yy” pattern where xx is the source language code and yy is the target language code. Default model for the pipeline is “t5-base”.  If you’re directly inferring with the T5 model, the model will expect a task prefix indicating the task itself, e.g. “translate: English to French”.
```

Comment:
To avoid confusion, I would remove the part of "If you're directly inferring with the T5 model".